StantUiIosLibrary/Classes/Headers/SecondHeader/SecondHeaderButton.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  SecondHeaderButton.swift
+//  StantUiIosLibrary
+//
+//  Created by Mac Mini Novo on 01/08/19.
+//
+
+import UIKit
+
+public struct SecondHeaderButton {
+    let action: Selector?
```

Comment:
Why is this optional if you are always creating it with a non-optional Selector?

Reply:
Because I forgot. Removed.